brightin-template

```diff
@@ -185,19 +186,21 @@ module BrightinTemplate
       copy_file 'coffee-script/coffeelint.rake', 'lib/tasks/coffeelint.rake'
     end
 
+    def install_npm_package
+      if npm? && options[:install]
+        system 'npm install --save-dev coffeelint'
+      else
+        say "You should install coffeelint globally:\n\n  npm install -g coffeelint"
+      end
+    end
+
```

Comment:
Couldn't find a method definition for `npm?` and at which `install_npm_package` is called during install, but I think I overlooked `include Helpers`. Never mind!